exchange/bidder.go

```diff
@@ -334,6 +347,7 @@ func (bidder *bidderAdapter) doRequest(ctx context.Context, req *adapters.Reques
 			}
 
 		}
+		bidder.me.engine.RecordAdapterFailedConnections(bidder.BidderName)
```

Comment:
I'm curious about this. A connection may fail due to our context deadline, which I don't think should be reported as a connection error. Are there other ways for `ctxhttp.Do` to error which has nothing to do with retrieving a connection?

Reply:
True. This metric will be removed per our conversation